### PR TITLE
Implement UI of group settings

### PR DESCRIPTION
**Pull request by ctiml**
- Add setting page
- Add info tab

Signed-off-by: Timothy Lee <ctiml@infuseai.io>

### CODE REVIEW

**Comment by ctiml** [comment]
Test image: `ch16089-670250e3`

**Comment by ctiml** [comment]
Test image: `ch16089-0daaa9b4`

**Comment by ctiml** [comment]
Test image: `ch16089-a5729520`

**Comment by ctiml** [comment]
Test image: `ch16089-0524d62a`

**Comment by ctiml** [comment]
Test image: `ch16089-dba11d56`

**Review by aar0nTw — CHANGES_REQUESTED** [review]

File: packages/client/src/components/groupSettings/alert.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Alert } from 'antd';
+
+export default class GroupSettingsAlert extends React.PureComponent {
+
+  render() {
+    return <Alert
+           message='Only system admins can update these settings'
```
Comment (aar0nTw) [inline packages/client/src/components/groupSettings/alert.tsx:8]: title: `These settings can be updated in Admin Portal`
desc: `Please update them in Admin Portal or contact the administrator.`

**Pull request merged**